src/plot/series/label-series.js

```diff
@@ -75,15 +83,19 @@ class LabelSeries extends AbstractSeries {
 
           const x = xVal + (allowOffsetToBeReversed && leftOfMiddle ? -1 : 1) * (xOffset || 0);
           const y = yVal + (allowOffsetToBeReversed && aboveMiddle ? -1 : 1) * (yOffset || 0);
+
+          const textAnchor = this.getTextAnchor(labelAnchorX, leftOfMiddle);
+          const alignmentBaseline = this.getAlignmentBaseline(labelAnchorY, aboveMiddle);
+
           const attrs = {
-            alignmentBaseline: aboveMiddle ? 'text-before-edge' : 'text-after-edge',
+            alignmentBaseline,
```

Comment:
you don't reuse textAnchor or alignmentBaseline, just call the functions when setting the attrs?